app/Http/Controllers/Api/CourseController.php

```diff
@@ -56,6 +59,55 @@ public function create(Request $request)
         return response()->json(['error' => 'This course already exist'], 409);
     }
 
+    public function deleteCourse(Request $request)
+    {
+        $courseId = $request->get('course_id');
+        $course = Course::find($courseId);
+        if($course){
+            $course->status = 'Inactive';
+            $save = $course->save();
+            if ($save) {
+                return response()->json(['success' => true], 200);
+            }
+        } else {
+            return response()->json(['error' => 'No course with this id'], 200);
+        }
+        return response()->json(['error' => 'something went wrong'], 400);
+    }
+
+    public function updateCourse(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'course_id' => 'required',
+        ]);
+
+        if ($validator->fails()) {
+            return response()->json(['error' => "course_id field is required"], 400);
+        }
+
+        $courseId = $request->get('course_id');
+        $course = Course::find($courseId);
+        if($course){
+            $request->get('title') != null ? $course->title = $request->get('title') : null;
+            $request->get('code') != null ? $course->code = $request->get('code') : null;
+            $request->get('description') != null ? $course->description = $request->get('description') : null;
+    
+            if(empty($request->get('title')) && empty($request->get('code'))){
+                return response()->json(['error' => 'All field is null'], 400);
+            } else {
+                $save = $course->save();
+            }
+    
+            if ($save) {
+                return response()->json(['success' => true], 401);
```

Comment:
Success should be in the 2XX code, basically 200 in this case not 401 or even 201. @bynalab